### PR TITLE
Add a comment about min version of OS X for USB logging

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: doxygen/main.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ DMX receiver / RDM responder.
 
 ## Common {#main-features-common}
 
-- \ref usb-logging, with adjustable log levels.
+- \ref usb-logging, with adjustable log levels. Requires OS X 10.7 or greater.
```
Comment (peternewman) [inline doxygen/main.md]: Can we word this in such a way it doesn't imply you need OS X. We should probably also clarify that Windows won't work without a driver file.

Comment (nomis52) [inline doxygen/main.md]: Done

File: doxygen/main.md [hunk 2]
```diff
@@ -40,7 +40,8 @@ DMX receiver / RDM responder.
 
 ## Common {#main-features-common}
 
-- \ref usb-logging, with adjustable log levels.
+- \ref usb-logging, with adjustable log levels. Requires Linux or MAC OS X 10.7 or
```
Comment (peternewman) [inline doxygen/main.md]: SPaG Mac

Comment (peternewman) [inline doxygen/main.md]: Re-ordering to list Mac or Linux would make it even clearer that the 10.7 requirement doesn't apply to Linux.

**Pull request merged**